src/guiKeyChangeMenu.cpp

```diff
@@ -354,6 +514,98 @@ bool GUIKeyChangeMenu::OnEvent(const SEvent& event)
 				case GUI_ID_ABORT_BUTTON: //abort
 					quitMenu();
 					return true;
+				case GUI_ID_KEY_ALIAS_ADD:
+					{
```

Comment:
Please remove the extra block. case behaves like goto labels, you don't need its own block.